scripts/e2e-build-package-publish.ts

```diff
@@ -12,7 +16,7 @@ export function buildPackagePublishAndCleanPorts() {
   removeSync('./tmp/angular/proj-backup');
   removeSync('./tmp/local-registry');
 
-  build(process.env.PUBLISHED_VERSION, '3.9.3', '2.1.2');
```

Comment:
Constant TypeScript and Prettier Version

Reply:
These should be updated to `4.2.4` and `2.2.1`, not dropped. The changed `build` function would replace versions in `create-nx-workspace` and `create-nx-plugin` to undefined. Please revert/update this.

Reply:
I know it is not clear from the changes view but the script now get's the versions from `@nrwl/workspace/src/utils/versions.ts`. The function parameters just had the same name as the imported constants from the version.ts file